src/main/java/seedu/address/logic/commands/task/todo/EditDeadlineCommand.java

```diff
@@ -28,12 +29,14 @@ public class EditDeadlineCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Edits the deadline of the specified todo from current available list.\n"
             + "Parameters: INDEX (INDEX must be a positive integer) "
-            + "[" + PREFIX_DEADLINE + "DEADLINE] (must be in format dd-mm-yyyy)\n"
+            + "[" + PREFIX_DEADLINE + "DEADLINE] (must be in format yyyy-mm-dd)\n"
```

Comment:
I actually find this a bit confusing, cuz in `InterviewDate` class, we are using a different format which is `MM/dd/yyyy hh:mm a`. Will it be more consistent to have a single format for all date-time? @jianminglok @benjamin-wee @zm-l how do yall think?